### PR TITLE
Xiaojian tang classifier1 translation

### PR DESCRIPTION
**Pull request by XiaojianTang**
Simplified Chinese Translation

### CODE REVIEW

File: 4-Classification/2-Classifiers-1/translations/README.zh-cn.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
+# 菜品分类器1
+
+本节课程将使用你在上一个课程中所保存的全部经过均衡和清洗的菜品数据。
+
+你将使用这份数据集，并通过多种分类器 _在给出了各种配料后预测这是那一个国家的菜品_。在此过程中，你将学到更多能够用来调试分类任务算法的方法。   
+
+## [课前测试](https://jolly-sea-0a877260f.azurestaticapps.net/quiz/21/)
```
Comment (mattholy) [inline 4-Classification/2-Classifiers-1/translations/README.zh-cn.md:7]: Pre-lecture quiz -> 课前测验

File: 4-Classification/2-Classifiers-1/translations/README.zh-cn.md [hunk 2]
```diff
@@ -0,0 +1,242 @@
+# 菜品分类器1
+
+本节课程将使用你在上一个课程中所保存的全部经过均衡和清洗的菜品数据。
+
+你将使用这份数据集，并通过多种分类器 _在给出了各种配料后预测这是那一个国家的菜品_。在此过程中，你将学到更多能够用来调试分类任务算法的方法。   
+
+## [课前测试](https://jolly-sea-0a877260f.azurestaticapps.net/quiz/21/)
+# 准备工作
+
+假设你已经完成了[课程1](../1-Introduction/README.md), 确保在根目录的`/data`文件夹中有 _cleaned_cuisines.csv_ 这份文件来进行接下来的四节课程。
```
Comment (mattholy) [inline 4-Classification/2-Classifiers-1/translations/README.zh-cn.md:10]: Assuming -> 假如
../1-Introduction/README.md -> ../../1-Introduction/translations/README.zh-cn.md

File: 4-Classification/2-Classifiers-1/translations/README.zh-cn.md [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+
+    ```python
+    from sklearn.linear_model import LogisticRegression
+    from sklearn.model_selection import train_test_split, cross_val_score
+    from sklearn.metrics import accuracy_score,precision_score,confusion_matrix,classification_report, precision_recall_curve
+    from sklearn.svm import SVC
+    import numpy as np
+    ```
+
+1. 接下来需要将数据分为训练模型所需的X（译者注：代表特征数据）和y（译者注：代表标签数据）两个dataframe。首先可将`cuisine`列的数据单独保存为的一个dataframe作为标签（label）。
```
Comment (mattholy) [inline 4-Classification/2-Classifiers-1/translations/README.zh-cn.md:44]: 有待进一步讨论

File: 4-Classification/2-Classifiers-1/translations/README.zh-cn.md [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+    ```output
+    0    indian
+    1    indian
+    2    indian
+    3    indian
+    4    indian
+    Name: cuisine, dtype: object
+    ```
+
+1. 调用`drop()`函数将 `Unnamed: 0`和 `cuisine`列删除，并将余下的数据作为可以用于训练的特证（feature）数据:
```
Comment (mattholy) [inline 4-Classification/2-Classifiers-1/translations/README.zh-cn.md:62]:
```suggestion
1. 调用`drop()`方法将 `Unnamed: 0`和 `cuisine`列删除，并将余下的数据作为可以用于训练的特证（feature）数据:
```

File: 4-Classification/2-Classifiers-1/translations/README.zh-cn.md [hunk 5]
```diff
@@ -0,0 +1,242 @@
...
+
+    ```python
+    cuisines_feature_df = cuisines_df.drop(['Unnamed: 0', 'cuisine'], axis=1)
+    cuisines_feature_df.head()
+    ```
+
+    你的特证（feature）数据看上去将会是这样:
```
Comment (mattholy) [inline 4-Classification/2-Classifiers-1/translations/README.zh-cn.md:69]:
```suggestion
    你的特征集看上去将会是这样:
```

File: 4-Classification/2-Classifiers-1/translations/README.zh-cn.md [hunk 6]
```diff
@@ -0,0 +1,242 @@
...
+    |      1 |        1 |     0 |          0 |     0 |            0 |       0 |        0 |         0 |         0 |    0 |     ... |           0 |          0 |                       0 |    0 |    0 |    0 |     0 |      0 |        0 | 0   |
+    |      2 |        0 |     0 |          0 |     0 |            0 |       0 |        0 |         0 |         0 |    0 |     ... |           0 |          0 |                       0 |    0 |    0 |    0 |     0 |      0 |        0 | 0   |
+    |      3 |        0 |     0 |          0 |     0 |            0 |       0 |        0 |         0 |         0 |    0 |     ... |           0 |          0 |                       0 |    0 |    0 |    0 |     0 |      0 |        0 | 0   |
+    |      4 |        0 |     0 |          0 |     0 |            0 |       0 |        0 |         0 |         0 |    0 |     ... |           0 |          0 |                       0 |    0 |    0 |    0 |     0 |      0 |        1 | 0   |
+
+现在，你已经准备好可以开始训练你的模型了！
+
+## 选则你的分类器
```
Comment (mattholy) [inline 4-Classification/2-Classifiers-1/translations/README.zh-cn.md:81]:
```suggestion
## 选择你的分类器
```

**Review by mattholy — CHANGES_REQUESTED** [review]
校对了一半，出现比较多的错误，建议仔细检查一下重新提交噢

**Comment by mattholy** [comment]
建议翻译前先参阅这个讨论组：
https://github.com/microsoft/ML-For-Beginners/discussions/201

**Comment by jlooper** [comment]
@mattholy does this look ok now? thanks!

**Review by mattholy — APPROVED** [review]

**Comment by mattholy** [comment]
> @mattholy does this look ok now? thanks!

That's pretty good now

**Review by feiyun0112 — CHANGES_REQUESTED** [review]
你将使用这份数据集，并通过多种分类器 在给出了各种配料后预测这是那一个国家的菜品。
->
您将使用此数据集和各种分类器，根据一组配料预测这是哪一国家的美食。

**Review by feiyun0112 — CHANGES_REQUESTED** [review]
../1-Introduction/README.md
->
../../1-Introduction/translations/README.zh-cn.md

**Review by feiyun0112 — CHANGES_REQUESTED** [review]
| almond | angelica | anise | anise_seed | apple | apple_brandy | apricot | armagnac | artemisia | artichoke |  ... | whiskey | white_bread | white_wine | whole_grain_wheat_flour | wine | wood |  yam | yeast | yogurt | zucchini |     |

->

| | almond | angelica | anise | anise_seed | apple | apple_brandy | apricot | armagnac | artemisia | artichoke |  ... | whiskey | white_bread | white_wine | whole_grain_wheat_flour | wine | wood |  yam | yeast | yogurt | zucchini |

**Review by feiyun0112 — CHANGES_REQUESTED** [review]
|          |        0 |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |

->

    |          |        0 |

**Review by feiyun0112 — CHANGES_REQUESTED** [review]
| precision    | recall | f1-score | support |      |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |

->

    |              | precision | recall | f1-score | support |

**Comment by XiaojianTang** [comment]
@feiyun0112 都改好了
All changed

**Review by feiyun0112 — APPROVED** [review]
ok

**Comment by jlooper** [comment]
thank you, merging!
